common/src/main/res/values/styles.xml

```diff
@@ -765,6 +765,10 @@
         <item name="android:textColor">@color/red_300</item>
     </style>
 
+    <style name="Subtitle2.14">
```

Comment:
You don't need to specify any styles with font sizes. Subtitle2 is derived from `TextAppearance.MaterialComponents.Subtitle2` which is already 14sp. Same for all the other design system styles.